src/click.rs

```diff
@@ -66,7 +71,7 @@ pub struct ClickConfigEntry {
     button: MouseButton,
     /// To which part of the block this entry applies
     #[serde(default)]
-    widget: Option<String>,
+    widget: Option<SerdeRegex>,
```

Comment:
Shouldn't this default to "block"?

Reply:
I decided to avoid unnecessary regex operations, so `None` is treated like `Some("block")`.

Reply:
Ah, I see that now